capa-controller-role/giantswarm-capa-role.tf

```diff
@@ -9,7 +9,7 @@ data "aws_iam_policy_document" "giantswarm-capa-controller" {
 
     principals {
       type        = "AWS"
-      identifiers = var.principal_arns_giantswarm_root_account
+      identifiers = "arn:aws:iam::${var.management_cluster_account_id}:user/${var.installation_name}-capa-controller"
```

Comment:
hey @pipo02mix , I think this is wrong (pinging you to confirm): The user is created on GS root account instead of the management cluster account. I think we should replace `management_cluster_account_id` with `principal_arns_giantswarm_root_account`.

Reply:
I think we had that discussion and I learnt that it works like that nowadays. I changed in the script but not here, thanks for catching this up